<table>
<colgroup>
<col style="width: 100%" />
</colgroup>
<thead>
<tr class="header">
<th><p><strong>The Francis Crick Institute</strong></p>
<p><strong>Advanced Sequencing Facility</strong></p>
<p><strong>Project Proposal Form</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td></td>
</tr>
</tbody>
</table>

<table>
<colgroup>
<col style="width: 48%" />
<col style="width: 51%" />
</colgroup>
<thead>
<tr class="header">
<th rowspan="2"><strong><u>Date</u></strong></th>
<th rowspan="2"><strong>21/01/2022</strong></th>
</tr>
<tr class="odd">
</tr>
</thead>
<tbody>
<tr class="odd">
<td rowspan="2"><strong><u>Project name</u></strong></td>
<td rowspan="2">Single-cell multiome ATAC and RNA sequencing of adult
hippocampal neural stem cells.</td>
</tr>
<tr class="even">
</tr>
<tr class="odd">
<td><strong><u>Investigator name</u></strong></td>
<td>Sara Ahmed de Prado</td>
</tr>
<tr class="even">
<td><strong><u>Investigator email</u></strong></td>
<td>saraahmeddeprado@crick.ac.uk</td>
</tr>
<tr class="odd">
<td><strong><u>PI / Lab name</u></strong></td>
<td><strong>Francois Guillemot</strong></td>
</tr>
<tr class="even">
<td><strong>Budget Code for sequencing work</strong></td>
<td><strong>10309</strong></td>
</tr>
<tr class="odd">
<td><strong><u>Is this from a grant or Core funded?</u></strong></td>
<td><strong>Wellcome Trust grant</strong></td>
</tr>
<tr class="even">
<td><strong><u>No. of samples planned for the project</u></strong></td>
<td>9</td>
</tr>
<tr class="odd">
<td><strong><u>Expected Date for Sample Submission</u></strong></td>
<td><p>6 weeks-old mice on 3<sup>rd</sup> February 2022 (Replicate
1)</p>
<p>12 months-old mice on 14<sup>th</sup> September 2022 (Replicate
1)</p>
<p>18 months-old mice on 5<sup>th</sup> September 2022 (Replicate
1)</p></td>
</tr>
<tr class="even">
<td><p><strong><u>Material to be submitted</u></strong></p>
<p><strong>Please provide as much detail as possible</strong></p></td>
<td>mCherry+ nuclei suspension resuspended in Diluted Nuclei Buffer
comprising 1X Nuclei buffer (10x Genomics), 1mM Dithiothreitol (DTT) and
1 U/ μL RNase inhibitor in nuclease-free water.</td>
</tr>
<tr class="odd">
<td><p><strong><u>Risk Assessment and Category Level Containment
Information</u></strong></p>
<p><strong>For all projects please state the containment level these
samples need to be handled at. Please also confirm an appropriate risk
assessment has been carried out for this work</strong></p></td>
<td><strong>CL1</strong></td>
</tr>
<tr class="even">
<td><strong><u>Type of Libraries</u> (e.g mRNAseq, ChIPseq,
Exome)</strong></td>
<td>Chromium Next GEM single cell multiome ATAC and RNA sequencing</td>
</tr>
<tr class="odd">
<td><strong>Sequencing Read Length (eg SR100, PE100)</strong></td>
<td>Paired-end, dual indexing</td>
</tr>
<tr class="even">
<td><strong>Organism</strong></td>
<td><em>Mus musculus</em></td>
</tr>
<tr class="odd">
<td rowspan="2"><strong>Any special run requirements/machine
type</strong></td>
<td rowspan="2"></td>
</tr>
<tr class="even">
</tr>
<tr class="odd">
<td></td>
<td></td>
</tr>
<tr class="even">
<td><p><strong>Number of reads per sample</strong></p>
<p><strong>Please be aware that this is expected to vary within a margin
of +/- 20 % per sample</strong></p></td>
<td></td>
</tr>
<tr class="odd">
<td></td>
<td></td>
</tr>
</tbody>
</table>

<table>
<colgroup>
<col style="width: 100%" />
</colgroup>
<thead>
<tr class="header">
<th rowspan="5"><p><strong>Project Summary</strong></p>
<p><strong>Introduction</strong></p>
<p>In mice, the subgranular zone of the hippocampal dentate gyrus has
lifelong neurogenic capabilities due to the presence of adult
hippocampal neural stem cells (NSCs or AHNSCs). Activation of NSCs
proliferation gives rise to neural progenitors, which will form the
dentate granule neurons upon differentiation, and/or new NSCs. In
absence of mitogenic signals, NSCs enter a non-dividing reversible state
or quiescence. As NSCs remain quiescent for longer duration, they move
progressively into deeper quiescence and display a reduced sensitivity
to growth signals. In juvenile mice, all proliferating hippocampal NSCs
differentiate into neuronal progenitors, whereas in adult mice (6 months
of age) more than half of the dividing NSCs self-renew and return to
quiescence.</p>
<p>Previous results from the lab demonstrate that the decision of adult
hippocampal NSC to re-enter the cell cycle or remain quiescent depends
on ASCL1 protein levels (Harris L. et al 2021). ASCL1 is a pioneer
transcription factor and thus it has the capacity to associate with
condensed chromatin to facilitate the binding of additional
transcription factors and it can directly modulate chromatin
accessibility, recruit other transcription factors and chromatin
modification enzymes. Other results from the lab also suggest that
significant transcriptional and chromatin accessibility changes occur
between activated and quiescent NSCs, based on previous snRNA-Seq (Fig.
1) analysis and preliminary snATAC-Seq analysis.</p>
<p><img src="proposal_media/image1.tiff"
style="width:3.50294in;height:2.33122in" /></p>
<p>Figure 1. single cell RNA-Seq analysis of adult hippocampal NSCs.
After two iterations of subsetting and re-clustering, a dataset of 2,947
NSCs was ordered using Slingshot, revealing a pseudotime trajectory from
the most quiescent NSCs (blue) to proliferating NSCs (red) (Harris L. et
al 2021).</p>
<p><strong>Hypothesis</strong></p>
<p>We propose that adult hippocampal NSC behavior is defined by their
epigenetic status, and thus the decision of adult hippocampal NSCs to
proliferate or remain quiescent is controlled cell intrinsically at the
chromatin level. In this model, quiescent NSCs from young mice would
present a partially closed chromatin configuration at loci involved in
NSC activity, and the chromatin configuration in these loci would be
closed more tightly as NSCs enter a deeper quiescence state in old
mice.</p>
<p><strong>Purpose</strong></p>
<p>Profiling of genome-wide chromatin accessibility and RNA expression
in the same individual adult hippocampal NSCs.</p>
<p>To explore the gene regulatory network that controls the maintenance
of quiescent AHNSCs and the progression of NSCs from quiescence to
activation, we plan to combine discovery of regulatory elements with
gene expression by simultaneous detection of mRNA and chromatin
accessibility from the same cell. The analysis of associations between
variation in gene expression and chromatin accessibility within the same
single cell will allow relationships to be identified unambiguously.</p>
<p>We will perform single cell multiome ATAC and RNA sequencing <em>in
vivo.</em> For this we will use a mouse strain generated by Dr. Kaja
Maczulska in which cells transcribing <em>Ascl1</em> express a
histone-tagged mCherry. To select the hippocampal NSCs we will use
FACS.</p>
<p>The scRNA-Seq data analysis will allow us to identify quiescent and
active NSCs and associate opened chromatin regions with active genes,
whereas the scATAC-Seq analysis will allow us to determine the
difference between quiescent and active NSCs at the chromatin level. We
propose that the chromatin accessibility level will correlate with the
position of cells along the quiescent to active axis.</p>
<p>Moreover, we plan to examine the role of TFs including ASCL1 and MYCN
in driving chromatin accessibility changes in NSCs by 1) examining
transcription factor binding sites in differential ATAC-Seq regions
between quiescent and active NSCs and 2) comparing the ATAC-Seq signal
with the CUT&amp;RUN signals for ASCL1 and MYCN generated in a parallel
project (PM22021). This will determine the role of TFs including Ascl1
and MycN in driving chromatin changes between quiescent and active NSCs,
and address the hypothesis that Ascl1 has distinct activities in
quiescent and active NSCs which we proposed in the above-mentioned
CUT&amp;RUN project proposal (PM22021).</p></th>
</tr>
<tr class="odd">
</tr>
<tr class="header">
</tr>
<tr class="odd">
</tr>
<tr class="header">
</tr>
</thead>
<tbody>
<tr class="odd">
<td><p><strong>Experimental Approach:</strong></p>
<p>Please include details such as the number of biological and / or
technical replicates:</p>
<p><strong>10x Genomics single cell multiome ATAC + gene expression
sequencing</strong></p>
<p>The experiments will be carried out with a mouse strain in which
cells transcribing <em>Ascl1</em> express a histone-tagged mCherry (Dr.
Kaja Maczulska). Multiomics (snRNA-Seq and snATAC-Seq) experiments will
be performed at three time points (6 weeks-old, 12 and 18 months-old
mice) with 4 mice of both males and females per age to obtain sufficient
NSCs. Three replicates will be generated per time point.</p>
<table>
<colgroup>
<col style="width: 42%" />
<col style="width: 27%" />
<col style="width: 30%" />
</colgroup>
<thead>
<tr class="header">
<th><strong>Age group</strong></th>
<th><strong>Technical Replicates</strong></th>
<th><strong>Biological Replicates</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td rowspan="3">6 weeks-old</td>
<td>Replicate 1</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="even">
<td>Replicate 2</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="odd">
<td>Replicate 3</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="even">
<td rowspan="3">12 months-old</td>
<td>Replicate 1</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="odd">
<td>Replicate 2</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="even">
<td>Replicate 3</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="odd">
<td rowspan="3">18 months-old</td>
<td>Replicate 1</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="even">
<td>Replicate 2</td>
<td>4 mice (males &amp; females)</td>
</tr>
<tr class="odd">
<td>Replicate 3</td>
<td>4 mice (males &amp; females)</td>
</tr>
</tbody>
</table>
<p><strong>Sample preparation</strong></p>
<p>All animal-related experimental procedures will be carried out under
ethical guidelines.</p>
<p>Mice will be killed by cervical dislocation and their brains removed.
Mouse dentate gyrus will be dissected, cut into small pieces and
transferred to a tube containing 300 μL ice cold NP40 nuclei lysis
buffer comprising 10 mM Tris-HCl pH 7.4, 10 mM NaCl, 3 mM MgCl2, 0.1%
Nonidet P40, 1mM DTT and 1 U/ μL RNase inhibitor in nuclease-free water.
The following procedure will be followed to extract the nuclei from the
dentate gyrus tissue: homogenization 15 times on ice cold nuclei lysis
buffer using a Pellet Pestle, incubation with 1 mL of ice-cold NP40
nuclei lysis buffer on ice for 5 min. Nuclei suspensions were passed
through a 70 μm Flowmi filter (VWR Bel-Art SP Scienceware). Then, nuclei
will be pelleted by centrifugation at 500rcf for 5 min at 4ºC leaving 50
μL of supernatant and adding 1mL of PBS 1% BSA 1 U/ μL RNase inhibitor
without mixing. After incubation on ice for 5 min, nuclei pellet willbe
resuspended and the nuclei suspension centrifugated at 500 rcf for 5 min
at 4ºC. Finally, nuclei pellet will be resuspended with 1mL of PBS 1%
BSA 1 U/ μL RNase inhibitor and 10 μL of 7-amino-actinomycin D (7AAD)
ready-made solution will be added to analyse viable cells in flow
cytometry.</p>
<p>To select only the nuclei of the cells that express <em>Ascl1</em>,
mCherry+ 7AAD+ nuclei will be sorted using a 100 μm nozzle and a flow
rate of 3 on a FACS Fusion A flow cytometer into a 5 mL FACS tube
containing 500 μL BSA with 5000U RNase inhibitor. After sorting, nuclei
concentration will be assessed using a hemocytometer/ the Countess II FL
Automated Cell Counter. The sorted nuclei will be pellet by
centrifugation at 500 rcf for 5 min at 4ºC, resuspended in 100 μL of
0.1X Lysis Buffer comprising 10 mM Tris-HCl pH 7.4, 10 mM NaCl, 3 mM
MgCl2, 0.1% Tween-20, 0.1% Nonidet P40, 0.01% Digitonin, 1% BSA, 1mM DTT
and 1 U/ μL RNase inhibitor in nuclease-free water and incubated on ice
for 2 min. Nuclei suspension will be mixed with 1 mL of Wash buffer
containing 10 mM Tris-HCl pH 7.4, 10 mM NaCl, 3 mM MgCl2, 0.1% Tween-20,
1% BSA, 1mM DTT and 1 U/ μL RNase inhibitor in nuclease-free water.
Nuclei will be pelleted by centrifugation at 500 rcf for 5 min at 4ºC
and resuspended in ice cold Diluted Nuclei buffer comprising 1X Nuclei
buffer (10x Genomics), 1mM DTT and 1 U/ μL RNase inhibitor in
nuclease-free water. Finally, nuclei concentration will be assessed
using a hemocytometer/ the Countess II FL Automated Cell
Counter.</p></td>
</tr>
<tr class="even">
<td></td>
</tr>
</tbody>
</table>

<table>
<colgroup>
<col style="width: 34%" />
<col style="width: 65%" />
</colgroup>
<thead>
<tr class="header">
<th colspan="2"><strong>Bioinformatics Analysis Requirements – please
complete this section in as much detail as possible</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Bioinformatics support required</strong></td>
<td><em>Yes</em></td>
</tr>
<tr class="even">
<td rowspan="2"><strong>Budget code for analysis work</strong></td>
<td rowspan="2"><strong>Specifically, the code for BABS to charge
against.</strong></td>
</tr>
<tr class="odd">
</tr>
<tr class="even">
<td rowspan="3"><strong>Analysis goals</strong></td>
<td rowspan="3"><p><strong>Please provide an outline of the goals of the
analysis</strong>:</p>
<p><em>Please provide a bullet point list of the analysis objectives and
questions that these data should address.</em></p>
<ul>
<li><p>Identification of cell types and assign cell-type identities to
snATAC-Seq clusters.</p></li>
<li><p>Identification of quiescent and activated states of
NSCs.</p></li>
<li><p>Compare chromatin accessibility in quiescent NSCs and active NSCs
within each aged group to determine the difference between quiescent and
active NSCs at the chromatin level.</p></li>
<li><p>Correlation between ATAC peak accessibility and gene expression
levels.</p></li>
<li><p>Compare snATAC-Seq and snRNA-Seq data to ASCL1 and MYCN binding
sites from CUT&amp;RUN project (PM22021) to identify binding sites
potentially involved in gene expression and chromatin opening.</p></li>
</ul>
<p><em>These objectives should be specific deliverables that constitute
the end-points for this project</em></p></td>
</tr>
<tr class="odd">
</tr>
<tr class="even">
</tr>
<tr class="odd">
<td><strong>Analysis details</strong></td>
<td><p><strong>Please provide details of the data analysis
required.</strong> </p>
<p><em>Please list all experimental factors for consideration, e.g.
genotype, phenotype, cell-line, treatment, batch, replicate
etc.</em></p>
<p>Experimental factors for the analysis:</p>
<ul>
<li><p>Replicates: 3 replicates per age group.</p></li>
<li><p>Age: young adults, middle-age adults and old adult mice (6
weeks-old, 12 and 18 months).</p></li>
</ul>
<p><em>Please provide all sample information in a separate Excel sheet
to be submitted to ASF and BABS in the sample sheet. Any factor names
and labels defined above should match those in the sample
sheet.</em></p>
<p><em>Please use these factors to list group comparisons to run e.g
treatment vs control, tumour vs normal, ChIP vs input etc. and the
definitions of any controls and how they are applied. In the case of
scRNA-Seq the characterization of specific cell types with marker genes
should be provided along with any trajectory analysis
requirements.</em></p>
<p><em>Any anticipated follow-up analyses including the integration of
other datasets such as previously analysed or public datasets should
also be defined here.</em></p>
<p><em>If a specific analysis technical is being applied beyond the
standard analysis workflows please provide details of the approach, a
link to software and any relevant
publications/documentation.</em></p></td>
</tr>
</tbody>
</table>

| **<u>BABS time estimate:</u>** |     |
|--------------------------------|-----|
|                                |     |
| **<u>ASF Cost Estimate:</u>**  |     |
|                                |     |
|                                |     |

Additional information from BABS:

**By submitting this form, you are confirming that your PI has agreed on
the project and that the cost-code can be used for this purpose.**

Please be aware that the number of hours listed to do the analysis is an
estimate and numbers may change. If the project takes longer than
estimated you will be informed, and a new estimation will require
approval before continuing. Likewise, if a project takes less time than
the initial estimate you will be charged accordingly.

This charge does not affect the [Crick's authorship
policy](https://intranet.crick.ac.uk/our-crick/research-integrity/pages/publication-authorship):
regardless of whether it is Core or Grant funded, we generally expect
our significant contribution to be recognised in papers - if this needs
discussion, please do so at the project proposal meeting.
